template/composer.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "zendframework/<package_name>",
+    "description": "<package_description, the same like the repository description>",
+    "homepage": "https://docs.zendframework.com/<package_name>/",
+    "license": "BSD-3-Clause",
+    "keywords": [
+        "zend",
+        "zendframework",
+        "<package_name>",
+        "<individual keywords for current package>"
+    ],
+    "support": {
+        "docs": "https://docs.zendframework.com/<package_name>/",
+        "issues": "https://github.com/zendframework/<package_name>/issues",
+        "source": "https://github.com/zendframework/<package_name>",
+        "rss": "https://github.com/zendframework/<package_name>/releases.atom",
+        "slack": "https://zendframework-slack.herokuapp.com",
+        "forum": "https://discourse.zendframework.com/c/questions/<components|expressive|apigility>"
+    },
+    "require": {
+        "php": "^5.6 || ^7.0"
```

Comment:
For new packages, this will be simply `^7.1`, in which case, we can change the PHPUnit constraint to simply `^6.2`.